kafka-ui-e2e-checks/src/main/java/com/provectus/kafka/ui/extensions/FileUtils.java

```diff
@@ -5,11 +5,26 @@
 import java.io.IOException;
 import java.nio.charset.StandardCharsets;
 
+import static org.apache.kafka.common.utils.Utils.readFileAsString;
+
 
 public class FileUtils {
 
-    public static String getResourceAsString(String resourceFileName) throws IOException {
-        return IOUtils.resourceToString("/" + resourceFileName, StandardCharsets.UTF_8);
+    public static String getResourceAsString(String resourceFileName) {
+        try {
+            return IOUtils.resourceToString("/" + resourceFileName, StandardCharsets.UTF_8);
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
please use logger

Reply:
Added logger

Reply:
refactor methods next way:

    public static String getResourceAsString(String resourceFileName) {
        try {
            return IOUtils.resourceToString("/" + resourceFileName, StandardCharsets.UTF_8);
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }

    public static String fileToString(String path) {
        try {
            return readFileAsString(path);
        } catch (IOException e) {
            throw new RuntimeException(e);
        }
    }

Reply:
no nullable needed, because in this case method should return non-null string
no logger is optional for try block

Reply:
@Haarolean fyi